pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -151,6 +150,7 @@ effectiveStdenv.mkDerivation rec {
   ]);
 
   nativeCheckInputs = lib.optionals pythonSupport (with python3Packages; [
+    gtest
```

Comment:
this should probably be moved outside of the pythonSupport flag as it doesn't exist under python package set.

Reply:
done

Reply:
> I've moved gtest back to its original place - it is only needed for the tests. 

Ugh there it goes again... so, why is it `nativeCheckInputs` and not `checkInputs`, and how does this work if we _hypothetically_ want to enable tests (using `hostPlatform.emulator`, presumably) in cross-compilation?

Reply:
Was like that before I started touching it 🤷 You're right though, it should probably go in `checkInputs`.

https://github.com/NixOS/nixpkgs/blob/3392575c4008b9a98dc69e68d02a031b379d5a81/pkgs/development/libraries/onnxruntime/default.nix#L129-L130

Reply:
I'm all sleepy, but this is probably relevant and I plan to acquaint tomorrow: https://github.com/NixOS/nixpkgs/pull/206742/files#diff-9538c800780031db3dfa7746f5a36fbbc895c60fdc896385bedd36940927427dR51